app.js

```diff
@@ -1,18 +1,20 @@
-const express = require('express');
+const express = require("express");
 const app = express();
+const getEndPoints = require("./controllers/api.controllers")
+const { getTopics } = require("./controllers/topics.controllers");
+app.get("/api", getEndPoints)
 
-const {getTopics} = require('./controllers/topics.controllers')
 
-app.get('/api/topics', getTopics)
+app.get("/api/topics", getTopics);
 
-app.use((err,req,res,next)=>{
-    res.status(500).send("Internal server error")
-    })
 
-    app.use((request, response) => {
-        response.status(404).send("Not found");
-      });
-  
 
-module.exports = app; 
+app.use((err, req, res, next) => {
+  res.status(500).send("Internal server error");
+});
 
+app.use((request, response) => {
```

Comment:
This isn't handling any errors, and infact this would never trigger so is therefore redundant. You also do not need to do any error handling for ticket 3 anyway so this a little weird to see here?

Reply:
If this was requested as part of a previous ticket then that's fine. However this isn't how you would implement error handling, also it would require testing